pkg/pillar/zedbox/zedbox.go

```diff
@@ -39,12 +41,17 @@ func main() {
 	switch basename {
 	case "client":
 		client.Run(ps)
+	case "command":
+		command.Run()
```

Comment:
Please pass it `ps` to keep with what the others do.

Reply:
Done